playbook-website/app/javascript/components/ReactSidebar/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import {Nav, NavItem, Caption} from 'playbook-ui'
+
+export const MENU_ITEMS = [
+  "Colors",
+  "Max Width",
+  "Number Spacing",
+  "Positioning",
+  "Line Height",
+  "Spacing",
+  "Border Radius",
+  "Typography",
+  "Shadows",
+  "Display",
+  "Cursor",
+  "Flex Box",
+];
+
+const ReactSidebar = () => {
```

Comment:
Could we just called this `Sidebar`? I don't know why we would need to add `React` to the naming unless I'm missing something.

Reply:
Will make that change 👍🏽